app/main.py

```diff
@@ -1 +1,8 @@
-# write your code here
+def copy_file(command: str) -> None:
+    if not command.startswith("cp"):
+        raise Exception("this interface support only 'cp' command")
+    ls_with_name = command.split()[1:]
```

Comment:
could u unpack like this 
`arg1, arg2, arg3 = command.split()` to avoid using indexes for better readability ?
also u can use underscores for useless values
f.e. `_, arg2, arg3 = command.split()`